camel/bots/discord_rag_bot.py

```diff
@@ -197,7 +200,7 @@ async def on_message(self, message: 'Message') -> None:
         await message.channel.send(response)
 
 
-async def process_message(agent: BotAgent, msg_queue: asyncio.Queue):
+async def process_message(agent: RagBotAgent, msg_queue: asyncio.Queue):
```

Comment:
My idea here is whether we can use the process_message function within the on_message event in the Discord app. Later, we would only need to modify process_message, making it easy to achieve the flow of Discord → message → custom process → response.